src/plugins/data/public/query/filter_manager/lib/mappers/map_phrase.ts

```diff
@@ -26,6 +26,11 @@ export function getPhraseDisplayValue(
   fieldType?: string
 ): string {
   const value = filter.meta.value ?? filter.meta.params?.query;
+
+  if (typeof value === 'string' && value === '') {
```

Comment:
You could use `hasValueToDisplay` that I introduced here: https://github.com/elastic/kibana/pull/121274

Reply:
Or add a helper `isEmptyString` to the same file